cms/static/js/views/tag_count.js

```diff
@@ -23,13 +23,12 @@ function($, _, BaseView, HtmlUtils) {
                     var data = event.data;
                     var courseAuthoringUrl = this.model.get("course_authoring_url")
                     if (event.origin == courseAuthoringUrl
-                        && data.includes('[Manage tags drawer] Count updated:')) {
+                        && data.type == 'authoring.events.tags.count.updated') {
```

Comment:
This is not working for me in tutor. event.origin is "http://apps.local.overhang.io:2001" while courseAuthoringUrl is "http://apps.local.overhang.io:2001/course-authoring".

Reply:
Changing this to `var courseAuthoringUrl = new URL(this.model.get("course_authoring_url")).origin;` worked.